assignments/array-methods.js

```diff
@@ -61,13 +61,12 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // Combine both the first and last names into a new array called fullName. 
 
 let fullName = [];
-console.log(fullName);
 
-runners.forEach(function(element){
-    fullName.push(element.first_name); 
-    fullName.push(element.last_name); 
-}); 
+runners.forEach(function(currentValue){
+    fullName.push(`${currentValue.first_name} ${currentValue.last_name}`); 
+});  
 
```

Comment:
Function works. You don't actually have to put function after your .forEach.

For example: 
runners.forEach(banana){
 fullName.push(`${banana.first_name} ${banana.last_name}`); 
}